README.md

```diff
@@ -35,3 +35,10 @@ View [example applications](/core/examples).
 ### Code formatting for contributors
 
 All projects in the repository use [Prettier](https://prettier.io) to format source code. Run the `npm install` command in the root folder to install it along with [husky](https://github.com/typicode/husky), the Git hooks manager. Both tools ensure proper codebase formatting before committing it.
+
+## Tests
+
+To test that existing features still work as expected after your changes, you can run UI tests from luigi-sample-angular example. To run tests, start the sample application by using `npm start` command in the application folder. When the application is ready:
```

Comment:
To ensure that existing features still work as expected after your changes, you can run UI tests from the 'luigi-sample-angular' example. To run the tests, start the sample application by using the `npm start` command in the application folder. 

When the application is ready: